application/akka/scala/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM mozilla/sbt:latest
+
+# Install SecretHub
+RUN curl -sLJO https://deb.secrethub.io/amd64
+RUN dpkg -i secrethub-cli-amd64.deb
+RUN rm secrethub-cli-amd64.deb
+
+# Create Akka Framework project template in Scala
+RUN sbt new akka/akka-scala-seed.g8 -Dsbt.rootdir=true --name=example --package=example 
```

Comment:
I know we do it like this in other examples too, but we do want to move away from templating in Dockerfiles. Could you run this outside of the Dockerfile and check the files into Git? So it'll be a more familiar Scala/Java project, with `src/main/scala/example/Example.scala`, etc.

Reply:
Done.